docs/guides/basics.rst

```diff
@@ -128,12 +128,7 @@ and will begin to pull the messages from the topic.
     from subs import photo_uploaded
 
     if __name__ == '__main__':
-        worker = Worker(
-            [photo_uploaded],
-            config.gc_project_id,
-            config.credentials,
-            config.ack_deadline,
-        )
```

Comment:
It's simpler but I'm trying to decide whether passing a single "config object" argument is better than explicit arguments. It could instead be that the broker/backend could be injected as an argument, itself initialized with the proper configuration.

Reply:
You could also inject the entire config object into the Worker which would have the backend initialized (like here).